helm/coco-kafka-bridge/app-configs/kafka-bridge_eks_delivery_prod_eu.yaml

```diff
@@ -32,19 +32,59 @@ bridges:
   replicas: 2
 #CMS metadata
 - name: cms-metadata-kafka-bridge-pub-prod-eu
-  sourceKafkaProxyUrl: "https://upp-prod-publish-eu.upp.ft.com"
+  sourceKafkaProxyUrl: "https://upp-prod-publish-eu.upp.ft.com/__kafka-rest-proxy"
   groupIdPrefix: metadata-kafka-bridge-pub-prod-eu
   topic: NativeCmsMetadataPublicationEvents
+  producer: "http://kafka-rest-proxy:8080"
   authSecretName: pub-auth
   authSecretKey: pub-prod-auth
   type: "proxy"
   replicas: 2
 - name: cms-metadata-kafka-bridge-pub-prod-us
-  sourceKafkaProxyUrl: "https://upp-prod-publish-us.upp.ft.com"
+  sourceKafkaProxyUrl: "https://upp-prod-publish-us.upp.ft.com/__kafka-rest-proxy"
   groupIdPrefix: metadata-kafka-bridge-pub-prod-us
   topic: NativeCmsMetadataPublicationEvents
+  producer: "http://kafka-rest-proxy:8080"
   authSecretName: pub-auth
   authSecretKey: pub-prod-auth
   type: "proxy"
   replicas: 2
+# Kafka -> MSK
+- name: kafka-to-msk-combined-post-publication-events-bridge
+  sourceKafkaProxyUrl: "http://kafka-rest-proxy:8080"
+  groupIdPrefix: kafka-msk-CombinedPostPublicationEvents-bridge
+  topic: CombinedPostPublicationEvents
+  producer: "http://kafka-rest-proxy-msk:8080"
+  authSecretName: pub-auth
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏Wondering do we need to set `authSecretName` and `authSecretKey` for the `Kafka -> MSK` and `MSK -> Kafka` cases because those seem to be credentials for the Publishing cluster?

Reply:
Great catch! Those are only used for the consumer configuration and might not be required at all. I will do some testing and report back the results.